huaweicloud/services/ga/resource_huaweicloud_ga_accelerator.go

```diff
@@ -484,7 +484,7 @@ func resourceAcceleratorUpdate(ctx context.Context, d *schema.ResourceData, meta
 	return resourceAcceleratorRead(ctx, d, meta)
 }
 
-func buildUpdateAcceleratorBodyParams(d *schema.ResourceData, config *config.Config) map[string]interface{} {
+func buildUpdateAcceleratorBodyParams(d *schema.ResourceData, _ *config.Config) map[string]interface{} {
```

Comment:
it's better to remove the second parameter

Reply:
fixed